packages/lime-system/files/usr/lib/lua/lime/utils.lua

```diff
@@ -57,18 +61,25 @@ function utils.applyHostnameTemplate(template)
 	return template:gsub("%%H", system.get_hostname())
 end
 
+function utils.get_id(string)
+	local id = {}
+	local fd = io.popen('echo "' .. string .. '" | md5sum')
+	if fd then
+		local md5 = fd:read("*a")
+		id[1] = md5:match("^(..)")
```

Comment:
It's a bit perverse to use match instead of for + strsub i see this thing was already present in our code but it's not good motivation to keep doing it wrong (it could start smell like chicken at some point )... in this cases, morover why not exposing the full md5 instead of only 6 couples?

Reply:
@G10h4ck less chicken taste now? https://github.com/libremesh/lime-packages/pull/349/files#diff-bf0ed9c5850e447bab27e71c3db02c03R70